cql3/functions/functions.hh

```diff
@@ -112,6 +110,9 @@ public:
 
     // Commit allows to atomically apply changes to main functions instance.
     void commit();
+
+    // Used only by unittest.
```

Comment:
this may change in the future and the comment would get obsoleted, maybe better to remove it, it doesn't introduce anything more than what can be looked up in the source code already

Reply:
that's the point that function which was added for unittest should not be used to anything else accidentally

Reply:
OK, then maybe:
```suggestion
    // Should be used only by unittest.
```
would be more explanatory.

Reply:
why?

Reply:
Because I interpreted what you wrote in the source code differently from what you wrote in reply to my comment, so if we want to avoid confusion, it's better to make code comments more detailed.

Reply:
but such rewrite should not change the meaning of the comment and you propose changing it